tarodb/asset_minting.go

```diff
@@ -332,6 +336,15 @@ func fetchAssetSeedlings(ctx context.Context, q PendingAssetStore,
 
 	seedlings := make(map[string]*tarogarden.Seedling)
 	for _, seedling := range dbSeedlings {
+		zeroKey := make([]byte, 32)
+		groupKey := &btcec.PublicKey{}
+		if !bytes.Equal(seedling.GroupKey[1:], zeroKey) {
```

Comment:
I agree, but I think the `AssetSeedling` type directly from the query has to be `[]byte`. I think for next version will check for an empty `[]byte` instead of a zero-valued array.

Reply:
Yes, with my comment above this can simply be a `len(seedling.GroupKey) > 0` check.